src/services/users.js

```diff
@@ -77,8 +79,40 @@ const signInUserByEmail = async (email, password) => {
 	return { token, userId: user.id }
 }
 
+/**
+ * service to invite user
+ * @param {string} email - email of the user to be invited
+ * @returns {object}
+ */
+const inviteUser = async (email) => {
+
+	const token = jwt.sign({ email }, 
+		config("TOKEN_SECRET"),
+		 {expiresIn: "7d"})
+
+	const url = `${frontendUrl}/signup?token=${token}`
+
+	const text = `Hi,
+	\n\nPlease click on the following link to complete your registration:
+	\n${url}\n\nIf you did not request this, please ignore this email.\n`
+
+	await emailService({ to: email, subject: "Invitation to signup", text })
+
+	const { rows, error } = await db.query(
```

Comment:
I see this in this particular trend in this file and I'm not sure it works that way.  

Reference here: https://node-postgres.com/api/result

 I know for sure we don't have to restructure the error and throw it ourselves. it will throw on its own